schema.js

```diff
@@ -0,0 +1,51 @@
+const YAML = require('js-yaml')
+const { flatten, includes, is, map, split, take } = require('ramda')
+
+const functionNames = [
+  'And',
+  'Base64',
+  'Cidr',
+  'Condition',
+  'Equals',
+  'FindInMap',
+  'GetAtt',
+  'GetAZs',
+  'If',
+  'ImportValue',
+  'Join',
+  'Not',
+  'Or',
+  'Ref',
+  'Select',
+  'Split',
+  'Sub'
+]
```

Comment:
ha! this is awesome!

Reply:
This could also be an external module since I already implement the short-hand syntax support for the framework with almost identical code (only lodash is replaced with ramda here) 😄

Reply:
oh @laardee! Your fingerprints are all over Serverless codebases! ❤️